2022/Days/day73.md

```diff
@@ -223,4 +223,4 @@ The DockerHub repository I used for this demo was a private one. But in the next
 - [GitHub Actions](https://www.youtube.com/watch?v=R8_veQiYBjI)
 - [GitHub Actions CI/CD](https://www.youtube.com/watch?v=mFFXuXjVgkU)
 
-See you on [Day 74](day74.md)
+See you on [Day 74](day74.md)
```

Comment:
@HieuChayA4 hey, can you get rid of this change? This file is unrelated to this PR, so keeping it as it is is better.

Reply:
I'm getting trouble resolve this , i copied the the original file but i keep put this to commit :(

Reply:
you can give git rebase or git reset a try next time :D